stubs/google/appengine/api/app_identity/__init__.pyi

```diff
@@ -0,0 +1,5 @@
+from typing import Optional, Text
+
+def get_application_id() -> Text: ...
+
+def get_default_gcs_bucket_name(deadline: Optional[float] = ...) -> Text: ...
```

Comment:
Why do we need this file? I don't think we use this library anymore.

Reply:
removed